pkg/storaged/client.js

```diff
@@ -1211,35 +1217,27 @@
     window.setInterval(stratis2_poll, 30000);
 }
 
-function stratis2_fixup_pool_notifications(data) {
+// When renaming a pool, stratisd 2.4.2 sends out notifications
+// with wrong interface names and forgets about notifications for
+// Devnode properties.
+//
+// https://github.com/stratis-storage/stratisd/issues/2731
+function stratis2_fixup_pool_notifications(path, props) {
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18361-20230216-212122-6af132ab-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)